api/src/main/java/marquez/db/DatasetDao.java

```diff
@@ -130,32 +147,51 @@ default void setFields(Dataset ds) {
   Optional<DatasetRow> getUuid(String namespaceName, String datasetName);
 
   @SqlQuery(
-      "SELECT d.*, dv.fields, sv.schema_location, facets, t.tags\n"
-          + "FROM datasets d\n"
-          + "INNER JOIN dataset_versions dv on d.current_version_uuid = dv.uuid\n"
-          + "LEFT OUTER JOIN stream_versions AS sv ON sv.dataset_version_uuid = dv.uuid\n"
-          + "LEFT OUTER JOIN (\n"
+      "WITH selected_datasets AS (\n"
+          + "    SELECT d.*\n"
+          + "    FROM datasets d\n"
+          + "    WHERE d.namespace_name=:namespaceName\n"
+          + "    ORDER BY d.name\n"
+          + "    LIMIT 10 OFFSET 0\n"
```

Comment:
@collado-mike I know this is already merged, but is this supposed to be `LIMIT :limit OFFSET :offset` instead?

Reply:
🤦🏽‍♂️

That's what I get for going back to my editor for one more test.

Reply:
We've all done it haha 😅  Speaking of tests, in mine the ORDER BY, LIMIT, AND OFFSET had to be at the end of the query in order for them to behave as expected (although I'm sure you noticed that too).